api/lib/domain/services/certification-candidates-ods-service.js

```diff
@@ -151,6 +152,7 @@ async function extractCertificationCandidatesFromCandidatesImportSheet({
 
       try {
         certificationCandidate.validate(isSco);
+        _checkForDuplication({ candidate: certificationCandidate });
```

Comment:
Pour 1 parametre, on est pas forcé de destructurer
```suggestion
	        _checkForDuplication(certificationCandidate);
```